src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -475,7 +500,6 @@ bool ov::pass::ConvertPrecision::run_on_model(const std::shared_ptr<ov::Model>&
         type_to_fuse[it.first] = it.second;
     }
 
```

Comment:
why was it deleted?

Reply:
No idea, unintentional change. Reverted.